cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Yes, that's true. Unfortunately, I realized that other than importing the reactors to the facility tag, I would have to actually deploy the reactors by adding the reactor names to the DeployInst tag, under institutions. 

So, the user who runs this script would have to add each reactors manually in facilities and add their names in DeployInst, which I thought, I could either use jinja and/or some script to make that process easier and less mistake prone.